webrtc.js

```diff
@@ -85,7 +85,7 @@ async function listAmendments() {
       throw new Error(`Amendments in container with id ${id} are mixing "modification" and "append" difftypes`);
     }
     if (amendments[id].some(a => a.type !== amendments[id][0].type)) {
-      throw new Error(`Amendments in container with id ${id} are mixing "corrections" and "addition" types`);
+      //throw new Error(`Amendments in container with id ${id} are mixing "corrections" and "addition" types`);
```

Comment:
@dontcallmedom : Do you want to merge this as is with the commented code?
Feel free to update if needed and then merge the PR.

Reply:
yes, this is intended, will merge